doc/quick_tutorial.rst

```diff
@@ -17,62 +17,108 @@
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
-.. _quick_tutorial_ref:
+.. Edited October 1, 2020 - [USA]TechDude
 
 Quick Tutorial
-==============
-
-Using OpenShot is very easy, and this tutorial will take you through the basics in
-**under 5 minutes**. After this tutorial, you will be able to make a simple photo
+===============
+OpenShot is easy to use, and it has many advanced features.  This introductory 
+tutorial shows you how to make a simple photo slide-show with music in 
+**less than 5 minutes**.  After this tutorial, you can make a simple photo 
 slide-show with music.
 
-Step 1 – Import Photos & Music
-------------------------------
-Before we can begin making a video, we need to import files into OpenShot. Drag and
-drop a few images (\*.JPG, \*.PNG, etc...) and a music file (most formats will work)
-from your Desktop to OpenShot. Be sure to drop the files where the
-arrow in the illustration is pointing to.
-
-.. image:: images/quick-start-drop-files.jpg
-
-Step 2 – Arrange Photos on Timeline
-------------------------------------
-Next, drag each photo onto the timeline (as seen in the illustration). The timeline represents
-your final video, so arrange your photos in whatever sequence you want them to appear in your
-video. If you overlap two clips, OpenShot will automatically create a smooth fade between them,
-displayed by blue rounded rectangles between the clips. Remember, you can rearrange the clips
-as many times as needed by simply dragging and dropping them.
+Basic Terminology
+-----------------
+To help understand the steps below, here are some definitions of a few basic 
+terms used in this Quick Tutorial.
+
+Clip:
+   A digitized or captured portion of video, audio, or both.  Clips are media 
```

Comment:
I love the idea of including definitions, but I dislike the actual definitions. 

Clip:
  Represents a media file at a specific place in time (position, start, end, layer)

Timeline:
  Represents a collection of clips, arranged by layer and time

Track:
  Represents a layer of video (higher layers cover up lower layers)

Transition:
  Represents a gradual change from one clip to the next one (fade in/out)